src/Store/FontStoreConnector.re

```diff
@@ -136,7 +130,7 @@ let start = (~getScaleFactor, ()) => {
             };
 
           let onError = errorMsg => {
-            Log.error("setFont: Failed to load font " ++ fullPath);
+            Log.error("Failed to load font " ++ fullPath);
 
             dispatch(
               ShowNotification(
```

Comment:
Kept despite `ShowNotification` including the same message, because the log level is error, which is quite significant.